webook/arrangement/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class UserHasNoPersonException(Exception):
+    pass
```

Comment:
Remember that we need to document this. start by doing it in the Wiki and also make a Kaizen PR to add a docstring to the exception.

Reply:
#222